components/kueue/development/kueue-external-admission/controller-patch.yaml

```
@@ -0,0 +1,5 @@
+- op: add
+  path: /spec/template/spec/containers/0/resources/limits
+  value:
+    cpu: 500m
+    memory: 512Mi
```

As this is a core component, we should enforce `Guaranteed` QoS. The following change should help:

```suggestion
- op: replace
  path: /spec/template/spec/containers/0/resources/requests
  value:
    cpu: 500m
    memory: 512Mi
- op: add
  path: /spec/template/spec/containers/0/resources/limits
  value:
    cpu: 500m
    memory: 512Mi
```